network/handlers/drain.go

```diff
@@ -38,6 +38,9 @@ func (s *sysTimer) tickChan() <-chan time.Time {
 	return s.Timer.C
 }
 
+// CheckOption enables configuration of an optional checker.
+type CheckOption func(w http.ResponseWriter, req *http.Request)
```

Comment:
This is `http.HandlerFunc`, so not strictly necessary.

Reply:
https://godoc.org/net/http#HandlerFunc

Reply:
Thanks for pointing out, removed.